packages/ui/v2/core/navigation/tabs/HorizontalTabItem.tsx

```diff
@@ -6,9 +6,12 @@ import { MouseEventHandler } from "react";
 import classNames from "@calcom/lib/classNames";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 
+import { PermissionContainer } from "../../PermissionContainer";
+
 export type HorizontalTabItemProps = {
   name: string;
   disabled?: boolean;
+  adminRequired?: boolean;
```

Comment:
If rather filter out in a parent container than adding logic to a dumb component

Reply:
true. i didnt see this